src/main/java/com/launchdarkly/client/RedisFeatureStore.java

```diff
@@ -278,10 +348,26 @@ public boolean initialized() {
    */
   public void close() throws IOException
   {
-    pool.destroy();
+    try {
+      if (executorService != null) {
+        executorService.shutdownNow();
+      }
+    } finally {
+      pool.destroy();
+    }
   }
 
-
+  /**
+   * Return the underlying Guava cache stats object.
+   *
+   * @return the cache statistics object.
+   */
+  public CacheStats getCacheStats() {
```

Comment:
I'm not too concerned about this-- seems useful for debugging tasks, and it's not part of the `FeatureStore` interface.